packages/kit/src/runtime/server/page/index.js

```diff
@@ -68,42 +50,22 @@ export async function render_page(event, route, page, options, state, resolve_op
 
 		let status = 200;
 
-		/** @type {HttpError | Error} */
-		let mutation_error;
+		/** @type {import('$app/forms').SubmissionResult | undefined} */
```

Comment:
I think `ActionResult` is probably a clearer name

```suggestion
		/** @type {import('$app/forms').ActionResult | undefined} */
```